tasks/deployment_manager/task.ts

```diff
@@ -152,6 +153,27 @@ task('migrate', 'Runs migration')
       );
       await dm.spider();
 
+      let governanceDm: DeploymentManager;
+      const base = env.config.scenario.bases.find(b => b.network === network && b.deployment === deployment);
```

Comment:
somewhat strange to pul this information from scenario bases, although `env.config.scenario.bases` is already a dependency of the `runMigration` task

Reply:
Yea... bases should probably be part of deployment manager, but its just a config namespace I guess? We could revisit separately